lumigator/python/mzai/backend/backend/tests/unit/api/routes/test_jobs.py

```diff
@@ -0,0 +1,45 @@
+import json
+import urllib
+from pathlib import Path
+
+from fastapi import status
+from fastapi.testclient import TestClient
+
+from backend.settings import settings
+
+
+def load_json(path: Path) -> str:
+    with Path.open(path) as file:
+        return json.load(file)
+
+
+def test_get_job_status(
+    app_client: TestClient,
+    job_repository,
+    request_mock,
+    json_ray_version,
+    json_data_health_job_metadata_ray,
+):
+    created_job = job_repository.create(name="test", description="")
+
+    print(f"RAY_VERSION: {settings.RAY_VERSION}")
+
+    request_mock.get(
```

Comment:
Can you please add a comment explaining why this is needed?
I think it'd be useful for ppl reading the code later on to understand that the ray client makes that call first, so we need to mock that up too